src/main/java/cz/muni/fi/PersonDao.java

```diff
@@ -0,0 +1,49 @@
+package cz.muni.fi;
+
+import java.util.List;
+
+/**
+ * Author: Richard Hrmo
+ */
+public interface PersonDao {
+
+    /**
+     * @return list of all drivers
```

Comment:
Pls rewrite all javadocs in this class. First there should short description of a method (capital letter at the beginning), afterwards single space followed by @param and @return tag if neccessary